qa/tasks/cephfs/kernel_mount.py

```diff
@@ -44,8 +44,8 @@ def mount(self, mntopts=[], check_status=True, **kwargs):
 
         self.setup_netns()
 
-        if not self.cephfs_mntpt:
-            self.cephfs_mntpt = '/'
+        #if not self.cephfs_mntpt:
```

Comment:
Now that this is done in _get_mount_cmd(), do you need this still?

Reply:
removed duplicate code
default `cephfs_mntpt` is now set in `qa/tasks/cephfs/mount.py::validate_subvol_options()`